### PR TITLE
Update readme file with the website content

### PR DESCRIPTION
**Pull request by izmalk**
## What is the goal of this PR?

Update the readme file to be in-line with the new typedb.com website.

## What are the changes implemented in this PR?

Copied most descriptions and code blocks from the Features page of Typedb.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,10 +340,31 @@ You can download TypeDB from the [Download Centre](https://vaticle.com/download)
...
+[Apache TinkerPop](http://tinkerpop.apache.org), 
+and [JanusGraph](http://janusgraph.org). 
+
+Thank you!
 
 ## Licensing
 
-This software is developed by [Vaticle](https://vaticle.com/).  It's released under the GNU Affero GENERAL PUBLIC LICENSE, Version 3, 19 November 2007. For license information, please see [LICENSE](https://github.com/vaticle/typedb/blob/master/LICENSE). Vaticle also provides a commercial license for TypeDB Cluster - get in touch with our team at commercial@vaticle.com.
+This software is developed by [Vaticle](https://vaticle.com/).  
+It's released under the GNU Affero GENERAL PUBLIC LICENSE, Version 3, 19 November 2007.
```
Comment (benjaminasmall) [inline README.md:366]:
```suggestion
It's released under the GNU Affero General Public License version 3 (AGPLv3) [maybe hyperlink?]. 
```

**Review by benjaminasmall — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,262 +8,302 @@
...
+sod-policy sub relation, 
+   owns name,
+   relates prohibits,
+   plays sod-violation:against;
 
+sod-violation sub relation,
+   relates of,
+   relates on,
+   relates against;
+```
```
[the remark this answers is not in the document]
Comment (izmalk) [reply README.md:107]: The main idea of this PR is to update the readme file with the content from our website. I really don't think we need to change examples here. I'm trying to be as close to our website as possible. And if we want to change our website example - we should probably do that as a separate task and later update this readme. What do you think?

File: README.md [hunk 3]
```diff
@@ -8,262 +8,302 @@
...
 
-## A higher degree of safety
+### Multi-valued attributes
 
-Types provide a way to describe the logical structures of your data, allowing TypeDB to validate that your code inserts and queries data correctly. Query validation goes beyond static type checking, and includes logical validations of meaningless queries. With strict type-checking errors, you have a dataset that you can trust.
+Use multi-valued attributes without having to create join tables or parse strings. If a person has multiple hobbies, 
+create multiple hobby attributes. And since attributes exist independently of entities and relations, and are shared, 
+there’s no need to create lookup tables. Simply query the database to find all attributes of a specific type
```
[the remark this answers is not in the document]
Comment (izmalk) [reply README.md:113]: This is much simpler but loses the comparison perspective. The comparison here is interesting and a useful one. Do you think we need to get rid of all comparisons here? Why?

File: README.md [hunk 4]
```diff
@@ -8,262 +8,302 @@
...
+   has full-name "John Doe", 
+   has primary-email "john.doe@vaticle.com",
+   has email-alias "jdoe@vaticle.com",
+   has email-alias "john@vaticle.com";
 ```
 
+## The ease of pattern matching
 
-### Logical Query Validation
+TypeQL is a fully declarative query language based on pattern matching. There is no need for developers to tell TypeDB
```
[the remark this answers is not in the document]
Comment (izmalk) [reply README.md:140]: Shane wrote this text. I'm not sure, but I'd assume Haikal reviewed it. But I totally see what you mean. The question is - whether we want to match Readme content with the website content (it was the original goal) or do better?

File: README.md [hunk 5]
```diff
@@ -8,262 +8,302 @@
...
-### Entity-Relationship Model
+TypeDB brings the benefits of strong typing in modern programming languages to the database, allowing developers to use 
+abstraction, inheritance and polymorphism when modeling and querying data – and finally removing the mismatch between 
+logical and physical data models.
 
-TypeDB allows you to model your domain using the well-known Entity-Relationship model. It is composed of entity types, relation types, and attribute types, with the introduction of role types. TypeDB allows you to leverage the full expressivity of the ER model, and describe your schema through first normal form.
+### Logical data model
+
+Model data naturally – as the entities, relations and attributes defined in an entity-relationship diagram. 
+There's no need for separate logical and physical data models due to database limitations and operational optimizations
```
[the remark this answers is not in the document]
Comment (izmalk) [reply README.md:26]:
```suggestion
There's no impedance mismatch between the logical and physical data models due to database limitations and operational optimizations.
```
Just moved a full stop here from the next line.
But can this phrase be seen as ambiguous?

File: README.md [hunk 6]
```diff
@@ -8,262 +8,302 @@
...
+TypeDB brings the benefits of strong typing in modern programming languages to the database, allowing developers to use 
+abstraction, inheritance and polymorphism when modeling and querying data – and finally removing the mismatch between 
+logical and physical data models.
 
-TypeDB allows you to model your domain using the well-known Entity-Relationship model. It is composed of entity types, relation types, and attribute types, with the introduction of role types. TypeDB allows you to leverage the full expressivity of the ER model, and describe your schema through first normal form.
+### Logical data model
+
+Model data naturally – as the entities, relations and attributes defined in an entity-relationship diagram. 
+There's no need for separate logical and physical data models due to database limitations and operational optimizations 
+(e.g., normalization). In TypeDB, the logical data model is the physical data model.
```
[the remark this answers is not in the document]
Comment (izmalk) [reply README.md:27]:
```suggestion
In TypeDB, the logical data model _is_ the physical data model.
```

**Pull request merged**